### PR TITLE
#168063210 View latest items first

### PR DESCRIPTION
**Pull request by rethes**
**What does this PR do?**

This PR implements the viewing of  latest articles and categories first

**Description of Task to be completed?**

- View latest articles and categories first

**How should this be manually tested?**

1. Navigate to ` News Articles or Categories`
2. Add an article and category
3. Should be listed first

**What are the relevant pivotal tracker stories?**

[#168063210](https://www.pivotaltracker.com/story/show/168063210)

**Screenshots (if appropriate)**

category
Screenshot 2019-08-28 at 20 33 27

article 
Screenshot 2019-08-28 at 20 32 18

**Questions:**

N/A

### CODE REVIEW

File: src/components/Categories/categories.vue [hunk 1]
```diff
@@ -55,25 +55,30 @@ export default {
...
-        post => Number(post.categoryId) === Number(selectedCategoryId),
+        post => Number(post.categoryId) === Number(selectedCategory.id),
       );
       this.remainingPosts.push(childPosts);
 
-      const index = this.categories.findIndex(category => category.id === selectedCategoryId);
+      const index = this.categories.findIndex(category => category.id === selectedCategory.id);
       this.categories.splice(index, 1);
-      this.$toastr.success('Category successfully deleted');
+      this.$toastr.success('deleted successfully', `${selectedCategory.title}`);
```
Comment (ndero) [inline src/components/Categories/categories.vue]: Nice use of template literals here.

File: src/components/common/postListContent.vue [hunk 2]
```diff
@@ -18,6 +24,9 @@
               <span class="user-name">
                 {{ post.author.name }}
               </span>
+              <span class="article-timestamps" >
+              . {{ postTimestamp( post.updatedAt) }}
```
Comment (ndero) [inline src/components/common/postListContent.vue]: The updated at timestamp is a great idea too

File: src/components/CategoryPosts/CategoryPosts.vue [hunk 3]
```diff
@@ -47,7 +50,12 @@ export default {
   methods: {
     // Get the posts for the particular category
     getPosts() {
-      return this.posts.filter(post => Number(post.categoryId) === Number(this.categoryId));
+      const categoryPosts = this.posts.filter(
+        post => Number(post.categoryId) === Number(this.categoryId),
```
Comment (ndero) [inline src/components/CategoryPosts/CategoryPosts.vue]: Why do you need to convert both `post.categoryId` and `this.categoryId` to number?

Comment (rethes) [reply src/components/CategoryPosts/CategoryPosts.vue]: Previously, the category id was of type number. 
But I will update this, as I changed the type to string.

**Pull request merged**